rpcs3/Emu/Cell/lv2/sys_rsx.h

```diff
@@ -117,6 +117,11 @@ struct RsxDisplayInfo
 	be_t<u32> pitch;
 	be_t<u32> width;
 	be_t<u32> height;
+
+	bool enabled() const
```

Comment:
I don't think you're allowed to disable an activated head though. Haven't tested on realhw but I expect unforseen issues with such a setup. Does sys_rsx allow this to happen?

Reply:
It happens in two situations:
- Registering buffer id which is greater than current display_buffer_count by more than one, so the buffer(s) inbetween are unitialialized and thus they have zero pitch/width/height.
- Registering it with such parameters, it is allowed to be done on the sys_rsx syscall although I think it would crash when trying to do a flip on it.

Reply:
That's what I was thinking as well. Maybe we should crash on trying to flip on it as well but that's an issue for another time.

Reply:
Renamed the methods to valid() as it seems more correct here, the check is currently only used in texture cache to check overlaps with existing framebuffer which is not flipping.